daemon/indexing/lib/search.js

```diff
@@ -113,16 +115,74 @@ class Listing {
         id: hit._id,
         name: hit._source.name,
         description: hit._source.description,
-        price: hit._source.price,
+        priceEth: hit._source.priceEth,
       }
       listings.push(listing)
     })
     return listings
   }
 }
 
+class Offer {
+  /*
+   * Indexes an Offerr
+   * @params {object} offer - JSON offer data from origin.js
+   * @throws Throws an error if indexing operation failed. 
+   */
+  static async index(offer, listing){
+    const resp = await client.index({
+      index: OFFER_INDEX,
+      type: OFFER_TYPE,
+      id: offer.id,
```

Comment:
Isn't offer id scoped to a listing ?
See https://github.com/OriginProtocol/origin-js/blob/master/contracts/contracts/marketplace/v01/Marketplace.sol#L161
Perhaps in order to generate a unique id for indexing purposes we could concatenate listing.id + offer.id ?

Reply:
*nods* Yes, this is the full, unique offer id, like "999-001-557-2"

Reply:
Perfect.